resources/l10n/ja/LC_MESSAGES/loot.po

```diff
@@ -1,424 +1,442 @@
+msgid "&Game"
+msgstr "ゲーム(&G)"
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:677
+msgid "&Edit Groups..."
+msgstr "グループの編集...(&E)"
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:679
+msgid "Searc&h Cards..."
+msgstr "カード検索...(&H)"
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:681
+msgid "Copy &Load Order"
+msgstr "ロードオーダーをコピー(&L)"
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:683
+msgid "&Copy Content"
+msgstr "データ内容をコピー(&C)"
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:685
+msgid "&Refresh Content"
+msgstr "データ内容を更新(&R)"
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:687
+msgid "Redate &Plugins..."
+msgstr "プラグインの日付変更...(&P)"
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:689
+msgid "&Fix Ambiguous Load Order"
+msgstr "不定なロードオーダーを修正(&F)"
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:691
+msgid "Clear All &User Metadata..."
+msgstr "ユーザーメタデータを完全に消去...(&U)"
+
+#. translators: The mnemonic in this string shouldn't conflict with other
+#. menus or sidebar sections.
+#: src/gui/qt/main_window.cpp:695
+msgid "&Plugin"
+msgstr "プラグイン(&P)"
+
+#. translators: This string is an action in the Plugin menu.
+#: src/gui/qt/main_window.cpp:697
+msgid "Copy &Plugin Name"
+msgstr "プラグイン名をコピー(&P)"
+
+#. translators: This string is an action in the Plugin menu.
+#: src/gui/qt/main_window.cpp:699
+msgid "Copy &Card Content"
+msgstr "カード内容をコピー(&C)"
+
+#. translators: This string is an action in the Plugin menu.
+#: src/gui/qt/main_window.cpp:701
+msgid "Copy &Metadata"
+msgstr "メタデータをコピー(&M)"
+
+#. translators: This string is an action in the Plugin menu.
+#: src/gui/qt/main_window.cpp:703
+msgid "&Edit Metadata..."
+msgstr "メタデータの編集...(&E)"
+
+#. translators: This string is an action in the Plugin menu.
+#: src/gui/qt/main_window.cpp:705
+msgid "Clear &User Metadata..."
+msgstr "ユーザーメタデータを消去...(&U)"
+
+#. translators: The mnemonic in this string shouldn't conflict with other
+#. menus or sidebar sections.
+#: src/gui/qt/main_window.cpp:709
+msgid "&Help"
+msgstr "ヘルプ(&H)"
+
+#. translators: This string is an action in the Help menu.
+#: src/gui/qt/main_window.cpp:711
+msgid "&View Documentation"
+msgstr "マニュアルを開く(&V)"
+
+#. translators: This string is an action in the Help menu.
+#: src/gui/qt/main_window.cpp:713
+msgid "&Join Discord Server"
+msgstr "Discordサーバーに参加(&J)"
+
+#. translators: This string is an action in the Help menu.
+#: src/gui/qt/main_window.cpp:715
+msgid "&About"
+msgstr "LOOTについて(&A)"
+
+#. translators: The mnemonic in this string shouldn't conflict with other
+#. menus or sidebar sections.
+#: src/gui/qt/main_window.cpp:720
+msgid "P&lugins"
 msgstr "プラグイン"
```

Comment:
You missed this mnemonic (&L).

Reply:
For those two, I did not add mnemonics because they are not strings for menu bar items and keyboard navigation does not work.
(An additional reason is that adding too many suffixes for mnemonics can look messy in Japanese.)
If there are consistency or maintenance issues with this, I'd prefer to remove all mnemonics from Japanese.

Reply:
No, that's fine, it just means that you can't switch between sidebar sections using the keyboard, but that's not such a big deal.

Reply:
Okay, I see. I don't think that is such a big deal either.
Thanks for the review!

Reply:
Thank you for your translations! If you wouldn't mind taking a look, there are three messages in the [prelude](https://github.com/loot/prelude) that are missing Japanese translations (i.e., `compatPatch`, `essentialFiles`, & `patchOutdated`).